lineman/app/services/thread_query_service.coffee

```diff
@@ -29,6 +30,7 @@ angular.module('loomioApp').factory 'ThreadQueryService', (Records) ->
     createTimeframeView = (name, filters, queryType, from, to) ->
       today = moment().startOf 'day'
       view = Records.discussions.collection.addDynamicView name
+      view.applyFind(groupId: {$in: CurrentUser.groupIds()})
```

Comment:
I think this limits the query service too much, as we'll want to be able to use it to display threads in groups you're not a part of at some point I reckon. Why not pass it in as a filter?